gadget/ondisk.go

```diff
@@ -366,6 +368,10 @@ func filesystemInfoForPartition(node string) (blk lsblkBlockDevice, err error) {
 	}
 }
 
+func toUpperUUID(bd *lsblkBlockDevice) {
```

Comment:
A New() constructor which would do this on creation might be preferrable, but if this is the only field that requires special treatment then maybe it's not worth it.

Reply:
Yeah eventually I would kind of like to move all these lsblk functions to the disks package instead where it's easier to mock, but for now I was just aiming for the minimal code change